CancerGov/_src/StyleSheets/_homelanding.scss

```diff
@@ -275,20 +276,58 @@ a.learn-more {
 	/* adding padding per #OCECREATIV-783 --sjc */
 }
 
-/* Class used to target homepage only with blue tint on 'Resources for' */
+/* Class previously used to target homepage only with blue tint on 'Resources for' */
 .guide-card--home {
-	div:first-child {
-		background: $lightblue;
+	.card {
+		display: flex;
+		flex-direction: column;
+	}
+	ul {
+		margin-bottom: 0;
+		display: flex;
+		flex-direction: column;
+		flex-grow: 2;
+		justify-content: flex-start;
 
-		h3, a, a.arrow-link-white {
-			color: $dgrey02;
+		a.arrow-link-white { /* targets 'Resources For' guide card to match 'About Cancer' li styling */
+		&::before {
+			@include svg-sprite(chevron-circle-lt-blue, $rotate:right);
+			}
+		&:hover::before {
+			@include svg-sprite(chevron-circle-red, $rotate:right);
+			  }
 		}
+		.card__view-more { /* targets last li to separate list from 'learn/view more' links  */
+			
+			padding: 3em 0 .5em; 
+			margin-bottom: 0;
 
-		a.arrow-link-white:hover {
-			@extend %card-link-hover;
+			a.arrow-link-white, a.arrow-link { /* targets last li item and adds dk blue chevron */
+				&::before {
+					@include svg-sprite(chevron-circle-dk-blue, $rotate:right);
+					}
+				&:hover::before {
+					@include svg-sprite(chevron-circle-red, $rotate:right);
+					  }
+			}
+			@include break(medium) {
+				padding: 2em 0 0;
+				margin-top: auto; /* align links to bottom */
+			}
+			@include break(tablet-ls) {
+				margin-top: auto;
+			}
+		
 		}
 	}
 }
+.guide-card--home .clearfix { 
+	flex-grow: 2;
```

Comment:
The previous comment applies here too. Is there any reason for this to be 2 and not just 1? Does it have an effect I'm not seeing?

Reply:
likewise, @alifr made this addition.

Reply:
also updated to flex-grow: 1